tools/run_danesfield.py

```diff
@@ -474,8 +478,8 @@ def main(args):
     cmd_args.append('--input_obj_paths')
     obj_list = glob.glob("{}/*.obj".format(roof_geon_extraction_outdir))
     # remove occlusion_mesh and results (building_<i>.obj)
-    obj_list = [e for e in obj_list
-                if e.find(occlusion_mesh) < 0 and e.find("building_") < 0]
+    #obj_list = [e for e in obj_list
+    #            if e.find(occlusion_mesh) < 0 and e.find("building_") < 0]
```

Comment:
Should use the same exclude buildings flag here (as described in the previous comment)

Reply:
we should check the contents of the output in `roof_geon_extraction_outdir`  I think there are some duplicate meshes at that stage, and this was added so that we don't include two copies in the rendering.

Reply:
There aren't duplicates in roof_geon_extraction. The building_ prefix is used to differentiate between road and building meshes. If meshes with that prefix are excluded then they're not processed by the texture_mapping and buildings_to_dsm steps.